app/src/test/java/mozilla/lockbox/presenter/ItemListPresenterTest.kt

```diff
@@ -183,15 +183,17 @@ open class ItemListPresenterTest {
 
     @Test
     fun `menuItem clicks cause RouteActions`() {
-        view.menuItemSelectionStub.onNext(R.id.fragment_setting)
+        view.menuItemSelectionStub.onNext(R.id.setting_menu_item)
+        dispatcherObserver.assertLastValue(RouteAction.SettingList)
+        view.menuItemSelectionStub.onNext(R.id.account_setting_menu_item)
         dispatcherObserver.assertLastValue(RouteAction.SettingList)
```

Comment:
@sashei I just noticed this test is failing now but this seems to fix it, right?

```suggestion
        dispatcherObserver.assertLastValue(RouteAction.AccountSetting)
```